src/test/mod.rs

```diff
@@ -548,8 +548,13 @@ impl TestFrontend {
         }
 
         debug!("loading template data");
-        let template_data =
-            Arc::new(TemplateData::new(&mut context.pool().unwrap().get().unwrap()).unwrap());
+        let template_data = Arc::new(
+            TemplateData::new(
+                &mut context.pool().unwrap().get().unwrap(),
+                context.config().unwrap().render_threads,
```

Comment:
Up in `base_config` we should probably set this to `1` to avoid spawning extra threads during testing.